### PR TITLE
Redesign main page

### PR DESCRIPTION
**Pull request by Hel5Sou**
Hi Dominick, these are my first main page changes. Let me know what you think. Thanks!

### CODE REVIEW

File: src/pages/login_page/login_page.scss [hunk 1]
```diff
@@ -14,10 +14,10 @@
...
 .form {
-  padding: 20px;
+  padding: 40px;
   flex-grow: 1;
-  display: inline;
+  display: inline;;
   text-align: center;
-  input[type="email"] { min-width: 200px; }
-  input[type="password"] { min-width: 200px; }
+  input[type="email"] {color: #1C1f2A; min-width: 300px; min-height: 25px; background-color: #b9dfeb; border-radius: 5px;}
```
Comment (dcaponi) [inline src/pages/login_page/login_page.scss]: Add color definitions to https://github.com/onelogin/onelogin-demo-lab/blob/redesign/src/styles/colors.scss

and give them descriptive names like $onelogin-blue so its easier to read the CSS after a few months.

File: src/ui_components/login_form/login_form.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ import SubmitButton from '../submit_button/submit_button';
 const LoginForm = (props) => {
   return (
     <form onSubmit={props.handleLogin}>
-      <input placeholder="Email" type="email" name="emailAddress"/><br/>
-      <input placeholder="Password" type="password" name="password"/><br/><br/>
+      <input id="Email" placeholder="Email" type="email" name="emailAddress"/><br/>
```
Comment (dcaponi) [inline src/ui_components/login_form/login_form.js]: Why did you add an #id here? Are you referencing it somewhere?

**Review by dcaponi — CHANGES_REQUESTED** [review]
Please see inline comments :)

File: src/pages/login_page/login_page.scss (hunk 1 above)
Comment (Hel5Sou) [reply src/pages/login_page/login_page.scss]: done :-) $onelogin-blue and $onelogin-black + I have add $onelogin-gray

File: src/ui_components/login_form/login_form.js (hunk 2 above)
Comment (Hel5Sou) [reply src/ui_components/login_form/login_form.js]: #id resolved and it's not there anymore

File: src/styles/colors.scss [hunk 3]
```diff
@@ -1,2 +1,14 @@
 $access-green: #1f9614;
 $off-white: #FFFFFE;
+$onelogin-blue: #00A9E0;
+$onelogin-black: #1C1F2A;
+$onelogin-gray: rgb(243, 243, 244);
+
+.onelogin-input {
```
Comment (dcaponi) [inline src/styles/colors.scss]: I think this needs to go to forms.scss. You just need colors definitions here

File: src/ui_components/header/header.scss [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .header {
-  background: $access-green;
-  color: $off-white;
-  padding: 10px;
+  background: #1C1F2a;
```
Comment (dcaponi) [inline src/ui_components/header/header.scss]: once you've defined $onelogin-black in the colors.scss file you should use the value here.

for example

.header {
  background: $onelogin-black;
}

**Review by dcaponi — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 5]
```diff
@@ -9,7 +9,7 @@ services:
       - testbed
     volumes:
       - .:/usr/src/app
-      - /usr/src/app/node_modules
+      - ./node_modules:/usr/src/app/node_modules
```
Comment (dcaponi) [inline docker-compose.yml:12]: How did you realize you needed to add this volume mount? Adding this breaks for me since I have no node_modules on my host, so I think you're running `npm install` on your side and then loading the node_modules into the container. 

Is there any reason for doing this over having node_modules live only inside the container?

Comment (Hel5Sou) [reply docker-compose.yml:12]: volume - yes, I'm now running it by using the npm run dev

Comment (Hel5Sou) [reply docker-compose.yml:12]: if you don't do the mount, you will have to rebuild the image everytime. Otherwise the change in new package will not be added to the image. So in order to make it working in dev mode and be able to make changes in modules, one needs to mount the node_modules

Comment (dcaponi) [reply docker-compose.yml:12]: That makes sense. I didn't see myself modifying packages often enough to care that I needed to rebuild each time and to be totally honest, this was just one of the tips I found on the internet for using docker in a react project so I wasnt sure what's more realistic.

We'll keep this change then. Do the `npm install` on the host and mount the `node_modules` in the container

File: src/pages/login_page/login_page.scss [hunk 6]
```diff
@@ -14,10 +14,9 @@
 }
 
 .form {
-  padding: 20px;
+  padding: 40px;
   flex-grow: 1;
-  display: inline;
+  display: inline;;
```
Comment (dcaponi) [inline src/pages/login_page/login_page.scss]: extra ;

File: src/pages/signup_page/signup_page.scss [hunk 7]
```diff
@@ -18,6 +18,7 @@
   flex-grow: 1;
   display: inline;
   text-align: center;
-  input[type="email"] { min-width: 200px; }
-  input[type="password"] { min-width: 200px; }
+  input[type="email"] { min-width: 300px; };
```
Comment (dcaponi) [inline src/pages/signup_page/signup_page.scss]: I don't think the ; after the } is needed

File: src/styles/colors.scss [hunk 8]
```diff
@@ -1,2 +1,6 @@
 $access-green: #1f9614;
 $off-white: #FFFFFE;
+$onelogin-blue: #00A9E0;
+$onelogin-black: #1C1F2A;
+$onelogin-gray: rgb(243, 243, 244);
```
Comment (dcaponi) [inline src/styles/colors.scss]: Did you want to use #F3F3F4 instead of rgb(243, 243, 244)

If you're interested in how they're the same https://www.rapidtables.com/convert/number/how-decimal-to-hex.html

File: src/styles/colors.scss (hunk 3 above)
Comment (Hel5Sou) [reply src/styles/colors.scss]: done

File: src/ui_components/header/header.scss (hunk 4 above)
Comment (Hel5Sou) [reply src/ui_components/header/header.scss]: done

File: src/pages/login_page/login_page.scss (hunk 6 above)
Comment (Hel5Sou) [reply src/pages/login_page/login_page.scss]: done

File: src/pages/signup_page/signup_page.scss (hunk 7 above)
Comment (Hel5Sou) [reply src/pages/signup_page/signup_page.scss]: true, you just need it after () in scss

File: src/styles/colors.scss (hunk 8 above)
Comment (Hel5Sou) [reply src/styles/colors.scss]: I have been using exactly what was in the onelogin marketing manual, but this makes sense - keep it with same "color - group" :-) ((https://www.rgbtohex.net/))

File: src/pages/signup_page/signup_page.scss (hunk 7 above)
Comment (Hel5Sou) [reply src/pages/signup_page/signup_page.scss]: done and true!

File: src/styles/colors.scss (hunk 8 above)
Comment (dcaponi) [reply src/styles/colors.scss]: Generally it doesn't matter a whole lot. I typically won't use rgb unless I need to specify opacity (thats actually rgba) hex value is just a little old school. Good to go for consistency though. 😄

**Review by dcaponi — APPROVED** [review]
